cdap-cli/src/main/java/co/cask/cdap/cli/command/CallServiceCommand.java

```diff
@@ -111,8 +119,8 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
 
   @Override
   public String getPattern() {
-    return String.format("call service <%s> <%s> <%s> [headers <%s>] [body <%s>] [body:file <%s>]",
-                         ArgumentName.SERVICE, ArgumentName.HTTP_METHOD,
+    return String.format("call service <%s> [version <%s>] <%s> <%s> [headers <%s>] [body <%s>] [body:file <%s>]",
```

Comment:
I think it has no ambiguition since there is a required `version` within the optional parameter